panther_analysis_tool/rule.py

```diff
@@ -512,13 +635,16 @@ def _get_destinations(
                     str(invalid_destinations),
                 )
                 return None
-            raise ValueError("Invalid Destinations: {}".format(str(invalid_destinations)))
+            raise UnknownDestinationError(
```

Comment:
Raising a specific exception type, helps to identify the error with precision and run the fallback return value check in PAT, when no available destination names are given through the CLI.